eva-accession-clustering/src/main/java/uk/ac/ebi/eva/accession/clustering/parameters/InputParameters.java

```diff
@@ -56,4 +61,19 @@ public int getChunkSize() {
     public void setChunkSize(int chunkSize) {
         this.chunkSize = chunkSize;
     }
+
+    public boolean isForceRestart() {
+        return forceRestart;
+    }
+
+    public void setForceRestart(boolean forceRestart) {
+        this.forceRestart = forceRestart;
+    }
+
+    public JobParameters toJobParameters() {
+        return new JobParametersBuilder()
+                .addString("assemblyAccession", assemblyAccession)
+                .addLong("chunkSize", (long) chunkSize, false)
+                .toJobParameters();
```

Comment:
why not adding the project accession and the vcf? if we want to rerun a failed job we can provide those same parameters, and if we want another job for another study, the project accession will be used to know it's a different job